content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
+1. [KKP Architecture, terminology and planning](../../architecture/)
+1. [Hardware requirements, firewall requirements](../../architecture/requirements/cluster-requirements/) and [supported cloud providers](../../architecture/supported-providers/) and [DNS requirements](../../installation/install-kkp-ce/#update-dns--tls)
+
+We will install KKP along the way, so, we do not need a running KKP installation. But if you already have KKP installation running, you can still make use of this guide to onboard your existing KKP installation to ArgoCD. This might involve in directory re-organization, though.
+
+## Introduction
+
+Below diagram shows the general concept of the setup. It shows how KKP installations and ArgoCD will be deployed and what KKP components will be managed by ArgoCD in each seed.
+![Concept](@/images/tutorials/gitops-argocd/kkp-gitops-argocd.png "Concept - KKP GitOps using ArgoCD")
+
+For the demonstration, 
+1. We will use 2 kubernetes clusters in AWS (created using Kubeone but they could be any Kubernetes clusters on any of the supported cloud / on-prem providers as long as they have a network path to reach each other)
+1. Install KKP master on one cluster (c1) and also use this cluster as seed (master-seed combo cluster)
+1. Make 2nd cluster (c2) as dedicated seed
+1. ArgoCD will be installed in each of the seed (and master/seed) to manage the respective seed's KKP components
+
+A high-level procedure to get ArgoCD to manage the seed would be as follows:
+1. Setup a kubernetes cluster to be used as master (or master-seed combo)
+1. Install ArgoCD helm chart and KKP ArgoCD Applications helm chart
+1. Install KKP on the master seed using kubermatic-installer
+1. Setup DNS records and Git repository tag so that ArgoCD can sync
+1. Sync the applications
+1. Repeat the procedure for 2nd seed (except installation of KKP on the seed)
+1. Add kkp seed record in master seed config files so that master takes control of seed cluster.
+1. Commit, push and re-tag the above change so that you can add the seed to master via ArgoCD UI
+
+Folder and File Structure section in the [README.md of ArgoCD Apps Component](https://github.com/kubermatic/kubermatic/blob/main/charts/gitops/kkp-argocd-apps/README.md) explains what files should be present for each seed in what folders and how to customize behavior of ArgoCD apps installation.
+
+**Note:** Configuring values for all the components of KKP is a humongous task. Also - each customer might like a different directory structure to manage KKP installation. This ArgoCD Apps based approach is an _opinionated attempt_ to provide a standard structure that can be used in most of the customer installations. If you need different directory structure, refer to [README.md of ArgoCD Apps Component](https://github.com/kubermatic/kubermatic/blob/main/charts/gitops/kkp-argocd-apps/README.md) to understand how you can customize this, if needed.
```

Comment:
instead of "each customer" maybe phrase it like "each KKP installation" or "each KKP setup".

Reply:
Same for "most of the customer installations"